ansible-collection-microsoft-sql/ansible-collection-microsoft-sql.spec

```diff
@@ -54,159 +35,50 @@ BuildRequires: ansible >= 2.9.10
 %endif
 
 %if %{without ansible}
-# Empty command. We don't have ansible-galaxy.
-%define ansible_collection_build() tar -cf %{_tmppath}/%{collection_namespace}-%{collection_name}-%{version}.tar.gz .
-%else
-%define ansible_collection_build() ansible-galaxy collection build
-%endif
-
-%if %{without ansible}
+# We don't have ansible-galaxy.
 # Simply copy everything instead of galaxy-installing the built artifact.
-%define ansible_collection_install() mkdir -p %{buildroot}%{ansible_collection_files}/%{collection_name}; (cd %{buildroot}%{ansible_collection_files}/%{collection_name}; tar -xf %{_tmppath}/%{collection_namespace}-%{collection_name}-%{version}.tar.gz)
+%define ansible_collection_build_install() tar -cf %{_tmppath}/%{collection_namespace}-%{collection_name}-%{version}.tar.gz .; mkdir -p %{buildroot}%{ansible_collection_files}/%{collection_name}; (cd %{buildroot}%{ansible_collection_files}/%{collection_name}; tar -xf %{_tmppath}/%{collection_namespace}-%{collection_name}-%{version}.tar.gz)
```

Comment:
> I think you should follow the macros (`ansible_collection_install()`, `ansible_collection_build()`) from the Ansible package ( https://src.fedoraproject.org/rpms/ansible/blob/rawhide/f/macros.ansible ) instead of defining your own `ansible_collection_build_install()` which does not have, AFAIK, any prior art.

it does have prior art - https://src.fedoraproject.org/rpms/linux-system-roles/blob/rawhide/f/linux-system-roles.spec#_59

Reply:
Ah. I missed this change, I would not expect it in a PR called "Fix README rendering problems".